UPGRADE-4.0.md

```diff
@@ -19,7 +19,7 @@ See also the [diff code](https://github.com/sonata-project/SonataAdminBundle/com
 
 ## Block id
 
-If you have created a custom `AbstractBlockService` you must now implement the new constructor, because all blocks use the service id as the block id now. Because of that, the translation keys for the block ids have changed from `sonata.block.$NAME` to `sonata.block.service.$NAME`.
+If you have created a custom `AbstractBlockService` you must now implement the new constructor, because all blocks use the service id as the block id now. 
```

Comment:
```suggestion
If you have created a custom `AbstractBlockService` you must implement the new constructor, because all blocks use the service id as the block id. 
```